pkg/web_app/lib/src/x_ago.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:html';
```

Comment:
This seems to be a too small method to be moved into a separate file. Let's keep it in `hoverable.dart` (otherwise it requires license header).

Reply:
It's worth splitting and you'll not need to remember that there is a hoverable configuration for `.-x-ago` in some other places.